### ISSUE TITLE
getBlock API doc not accurate

### ISSUE THREAD

**Issue by softsimon** [mempool/mempool#2818]
The getBlock API docs is showing the the block `extras` properties like feeRange. But clicking on the link to the API the extras doesn't exist there.

This is because we have 2 block APIs
`/api/block` that goes to esplora and
`/api/v1/block` that goes to NodeJS and our database with the extras is present

Maybe both should be documented separately or just fix the current one.

**Comment by wiz** [comment]
AFAIK hunicus said he couldn't easily fix this when we raised the issue to him last week, so he's in the middle of rewriting the entire docs...

**Comment by hunicus** [comment]
Yes...once and for-all docs refactor is in progress. If I don't finish by the end of this week, I'll post a fix for this with a work-around.

**Comment by hunicus** [comment]
Update: I blew past my own due date on this but it's happening. Finished a new structure with much more configurability and automated response-fetching. 

Just need to do a lot of copy-pasting now to bring all the old content to new format...hopefully for the last time.

https://github.com/hunicus/mempool/tree/refactor-api-docs

**Comment by nymkappa** [comment]
Duplicate https://github.com/mempool/mempool/issues/3192
